moveit.rosinstall

```diff
@@ -32,4 +32,4 @@
 - git:
     local-name: octomap
     uri: https://github.com/OctoMap/octomap.git
-    version: v1.8.0
\ No newline at end of file
+    version: v1.8.0
```

Comment:
this change is unrelated, but it's caused by GitHub's web based file editor - I can removed it if it's unwanted though